metriq_gym/circuits.py

```diff
@@ -36,3 +46,206 @@ def qiskit_random_circuit_sampling(n: int) -> QuantumCircuit:
             t = unused_bits.pop()
             circ.cx(c, t)
     return circ
+
+
+def distribute_edges(graph: nx.Graph) -> list:
```

Comment:
We will need unit tests for those general functions

Reply:
done